include/multipass/virtual_machine_description.h

```diff
@@ -39,13 +41,14 @@ class VirtualMachineDescription
     MemorySize mem_size;
     MemorySize disk_space;
     std::string vm_name;
-    std::string mac_addr;
+    std::vector<NetworkInterface> interfaces;
```

Comment:
I considered to have one default interface and a list of extra interfaces, but I decided to keep all of them toghether because it's possible at some point to eliminate the distinction between the interfaces. This way, we won't change `VirtualMachineDescription` twice and, in the future, we will remove the special treatment to the first element of the vector.

Reply:
I separated the default network interface from the list of extra interfaces. This is resolved.